src/components/ConferenceItem/ConferenceItem.js

```diff
@@ -38,20 +38,30 @@ export default class ConferenceItem extends PureComponent {
             {name}
           </Link>
         </Heading>
-        <p className={styles.p}>
-          {`${Location(city, country)}・`}
-          <span className={styles.Date}>
-            {formatDate(startDate, endDate)}
-          </span>
-        </p>
-        <p className={classNames(styles.p, styles.Footer)}>
-          {showCFP && <Cfp url={cfpUrl || url} date={cfpEndDate} />}
-          {showCFP && <br />}
-          <Topics topics={topics} />
-          {twitter && ' – '}
-          <Twitter twitter={twitter} />
-        </p>
-      </div>
+        <dl className={styles.dl}>
+          <div>
+            <dt className="visuallyHidden">Location</dt>
+            <dd>
+              {`${Location(city, country)}`}
+            </dd>
+          </div>
+          <div>
+            <dt className="visuallyHidden">Date</dt>
+            <dd>
+              <span aria-hidden="true">・</span>{formatDate(startDate, endDate)}
+            </dd>
+          </div>
```

Comment:
Instead of using floats, can't we group the two divs on line 42 and 48 inside a common div and use `display: inline-block`? Same for the `divs` that are after.
Basically group each line within a common ancestor.

Reply:
I wish so bad. I tried everything I know with CSS grid, Flexbox, but had to stick with floats.

The reason is, we can't have two `dt/dd` inside one `div`.

This is invalid HTML : 

```HTML
<dl>
  <div>
    <dt>Term</dt>
    <dd>Description</dd>
    <dt>Term</dt>
    <dd>Description</dd>
  </div>
</dl>

Reply:
Han interesting, I had no idea.
Apparently if we add an empty div it could help but that's hackier so  let's go with float then!